packages/utils/src/cjs.js

```diff
@@ -1,6 +1,12 @@
+import { sep } from 'path'
+
+export function isExternalDependency (id) {
+  return id.includes(`${sep}node_modules${sep}`)
```

Comment:
I think it should cover **both** separators. A windows user can also call require with slash :)